modules/cognito/outputs.tf

```diff
@@ -7,5 +7,6 @@ output outputs {
     user_pool             = aws_cognito_user_pool.concourse
     user_pool_main_domain = aws_cognito_user_pool_domain.main
     issuer                = "https://cognito-idp.${local.region}.amazonaws.com/${aws_cognito_user_pool.concourse.id}"
+    concourse_pool        = aws_cognito_user_pool.concourse
```

Comment:
What does this do?  same as `user_pool` ?

Reply:
This is needed for metrics repo to have the resource added

Reply:
Oh I see. Yes removed.